qiskit/synthesis/multi_controlled/mcmt_vchain.py

```diff
@@ -42,6 +42,15 @@ def synth_mcmt_vchain(
         q_6: ─────┤ X ├──■────■──┤ X ├─────
                   └───┘          └───┘
 
+    Args:
+        gate: base gate to be applied to the targets.
+        num_ctrl_qubits: Number of control qubits.
+        num_target_qubits: Number of target qubits.
+        ctrl_state: Optional control state as an integer.
+
+    Returns:
+        QuantumCircuit: The synthesized circuit for the MCMT gate.
```

Comment:
We don't need the type hints here since it's already in the signature (same comment applies on the other docstring below) 🙂 
```suggestion
        The synthesized circuit for the MCMT gate.
```

Reply:
Fixed